src/app/Joyride/CryostatJoyride.tsx

```diff
@@ -0,0 +1,277 @@
+          content: (
+            <p>
+              Create, delete, enable, and view Cryostat <strong>Automated Rules</strong> in this view. Automated Rules
+              allow you start recordings on target JVMs based on a set of conditions.
+            </p>
+          ),
+          target: '*[data-tour-id="automatedrules"]',
+          placement: 'right',
+        },
+        {
+          title: 'JFR Recordings',
+          content: (
+            <p>
+              The <strong>Recordings</strong> view provides a list of all active recordings that are currently being
+              recorded on the target JVM. Start, stop, download, delete recordings from this view.
+            </p>
+          ),
+          target: '*[data-tour-id="recordings"]',
+          placement: 'right',
+        },
+        {
+          title: 'Archives View',
+          content: (
+            <p>
+              The <strong>Archives</strong> view provides a list of all saved recordings that have been saved to
+              Cryostat. Download, delete, and generate reports from this view.
+            </p>
+          ),
+          target: '*[data-tour-id="archives"]',
+          placement: 'right',
+        },
+        {
+          title: 'Events',
+          content: (
+            <p>
+              The <strong>Events</strong> page lists the <strong>Event Templates</strong> that can be used for creating
+              Flight Recordings. It also details the JFR <strong>Event Types</strong> that can be recorded within each
+              target JVM.
+            </p>
+          ),
+          target: '*[data-tour-id="events"]',
+          placement: 'right',
+        },
+        {
+          title: 'Security',
+          content: (
+            <p>
+              The <strong>Security</strong> tab allows you to add <strong>Credentials</strong> and{' '}
+              <strong>SSL Certificates</strong> for Cryostat to use when connecting to remote targets.
+            </p>
+          ),
+          target: '*[data-tour-id="security"]',
+          placement: 'right',
+        },
+        {
+          title: 'Settings',
+          content: (
+            <p>
+              Set your <strong>Cryostat preferences</strong>, such as the theme, locale, notification settings, and
+              more.
+            </p>
+          ),
+          target: '*[data-tour-id="settings-link"]',
+        },
+        {
+          title: 'Help',
+          content: (
+            <p>
+              Restart this tour or access our new <strong>quick starts</strong> where you can learn more about using
+              Cryostat in your environment.
+            </p>
+          ),
+          target: '*[data-tour-id="application-launcher"]',
+        },
+        {
+          title: 'You’re ready to go!',
+          content: (
+            <p>
+              Stay up-to-date with everything Cryostat on our{' '}
+              <a target="_blank" href={`${build.homePageUrl}/blog/`} rel="noreferrer">
```

Comment:
I think this whole URL should be extracted to the `build.json` file, and same for any other cases (like a few lines lower) where a path is resolved against one of the keys from that file. It may be more than just the base of the URL that needs to be patched between an upstream and downstream build, ie. it may not be the case that a downstream patched version uses a URL with the path ending in `/blog` here.

Reply:
I added a key called 
```json
  "documentationUrl": "https://cryostat.io/guides",
```
 Something like `{build.documentationUrl}/#create-an-automated-rule` should be extracted to `build.json` as well like `build.createAutomatedRuleGuideUrl` ?

Reply:
For now let's just go with that. Again, it could be that we change the upstream website structure or there is a downstream patched build that needs a different structure, and maybe the guide section isn't a page anchor but a different on the server, for example.

If that ends up being a significant annoyance/maintenance burden with a lot of instances of copy-pasted relative URLs like that then we can look for a specific solution for that problem.